apps.toml

```
@@ -2513,6 +2513,13 @@ state = "working"
 subtags = [ "business_and_ngos" ]
 url = "https://github.com/YunoHost-Apps/ofbiz_ynh"
 
+[ombi]
+category = "multimedia"
+potential_alternative_to = [ "Plex", "Netflix" ]
+state = "working"
+subtags = [ "music", "mediacenter" ]
```

```suggestion
subtags = [ "mediacenter" ]
```

It does not seem to manage music.